COMET.Web.Common.Tests/Components/LoginTestFixture.cs

```diff
@@ -70,6 +73,36 @@ public void Teardown()
             this.context.CleanContext();
         }
 
+        [Test]
+        public async Task VerifyFocusingAndBluring()
+        {
+            var renderer = this.context.RenderComponent<Login>();
+
+            Assert.That(renderer.Instance.FieldsFocusedStatus, Is.EqualTo(new Dictionary<string, bool>()
+            {
+                { "SourceAddress", false },
+                { "UserName", false },
+                { "Password", false }
+            }));
+
+            Assert.IsFalse(renderer.Instance.FieldsFocusedStatus["UserName"]);
```

Comment:
Please use Assert.That and surround the two Assert inside a Assert.Multiple( () => ... );